medusa/server/web/home/handler.py

```diff
@@ -1293,8 +1293,8 @@ def check_show_for_language(show_obj, language):
         indexer_api_params = indexerApi(show_obj.indexer).api_params.copy()
         indexer_api_params['language'] = language
         indexer_api_params['episodes'] = False
-        t = indexerApi(show_obj.indexer).indexer(**indexer_api_params)
         try:
+            t = indexerApi(show_obj.indexer).indexer(**indexer_api_params)
```

Comment:
You aren't catching `IndexerException`, you are catching `IndexerShowNotFoundInLanguage`.  So if the creation of the indexer raises `IndexerException` then it won't be caught.  If you don't intend for it to be caught then there's no reason for the indexer creation to be inside the `try: except`.  If you do want it to be caught you need to change the `except` clause to `except IndexerException`.